content/howto/extensibility/how-to-create-a-native-pluggable-widget.md

```diff
@@ -0,0 +1,1344 @@
+---
+title: "Create a native pluggable widget"
+#category: "Enter the category under which the document should be published (for example, "Mobile"); if there is a parent, remove this category line"
+#parent: "Enter the parent document filename of this document if necessary (for example, "design-the-architecture"); if there is a category, remove this parent line"
+#menu_order: Enter the position of the document in the category or under the parent; number by 10 (for first), 20, 30, etc. for easy ordering of other documents in the future if necessary; don't add brackets or quotation marks; if no number is added, the system will add an extremely high number to order the documents, which means that if you only want a document to appear at the top, you only have to add "10" to that specific document, you don't have to order all the other documents in the category/under the parent
+#description: "Set a description with a maximum of 140 characters; this should describe what the goal of the document is, and it can be different from the document introduction; this is optional, and it can be removed"
+#tags: [Add a maximum of 5-7 tags/keywords; keep them focused on the most important topics of the document, and make sure the tag is used as a word in the actual content (will function best for SEO as a word in a heading); each tag should have quotation marks and be separated by a comma, for example: "Samba", "MxCloud", "cloud", "share"; the tags should be enclosed with brackets and quotation marks]
+---
+
+## 1 Introduction
+
+Pluggable widgets enable developers to introduce custom-built widgets in their Mendix applications. Not just for web, but also for native mobile. In this how-to you will learn to create a group box pluggable widget for native mobile applications.
+
+**This how-to will teach you how to do the following:**
+
+- Generate a widget structure
+- Create...
+- Build...
+- Configure...
+
+**Are you in a hurry?**
+
+Clone this [code sample](https://github.com/mendix/native-group-box-pluggable-widget-sample) from GitHub with all group box features from this how-to already implemented.
+
```

Comment:
image image for expected result (mobile and modeler screen shot)

Reply:
Images will be created by Connor.